Core/clim-basic/clipboard.lisp

```diff
@@ -0,0 +1,76 @@
+(in-package :climi)
+
```

Comment:
please explain in a toplevel comment what is clipboard, what is selection, how they interact with each other, what are differences and similarties. in other words provide these newly introduced concepts summary.

Reply:
I have added a comment section at the beginning of the file explaining some of the terminology.